src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift

```diff
@@ -51,6 +51,28 @@ class HomeStatisticsCardViewModel {
 		}
 	}
 	
+	init(administrativeUnitData: SAP_Internal_Stats_AdministrativeUnitData, district: String) {
+		let sevenDayTrend = administrativeUnitData.sevenDayIncidence
+		
+		title = AppStrings.Statistics.AddCard.localCardTitle
+		titleAccessibilityIdentifier = AccessibilityIdentifiers.LocalStatistics.localStatisticsCard
+		subtitle = district
+		illustrationImage = UIImage(named: "LocalIncidence")
+		primaryValue = sevenDayTrend.formattedValue
+		primaryTrendImage = sevenDayTrend.trendImage
+		primaryTrendImageTintColor = sevenDayTrend.trendTintColor
+		primaryTrendAccessibilityLabel = sevenDayTrend.trendAccessibilityLabel
+		primaryTrendAccessibilityValue = sevenDayTrend.trendAccessibilityValue
+
+		let updateDate = Date(timeIntervalSince1970: TimeInterval(administrativeUnitData.updatedAt))
+		primaryTitle = updateDate.formatted(
+			todayString: AppStrings.Statistics.Card.Infections.today,
+			yesterdayString: AppStrings.Statistics.Card.Infections.yesterday,
+			otherDateString: AppStrings.Statistics.Card.Infections.date
```

Comment:
```suggestion
			todayString: AppStrings.Statistics.Card.Incidence.today,
			yesterdayString: AppStrings.Statistics.Card.Incidence.yesterday,
			otherDateString: AppStrings.Statistics.Card.Incidence.date
```

Either use these or create new ones.